engine/engine/src/engine.cpp

```diff
@@ -857,6 +857,8 @@ namespace dmEngine
         dmSound::Result soundInit = dmSound::Initialize(engine->m_Config, &sound_params);
         if (dmSound::RESULT_OK == soundInit) {
             dmLogInfo("Initialised sound device '%s'", sound_params.m_OutputDevice);
+        } else {
+            dmLogWarning("Failed to initialize sound system.");
```

Comment:
We could ofc exit the engine.
But, if it's still possible to run the game without sound, it's probably better to try that instead.